File: phonebook.py
peaple=[
    {"name":"basem","number": "+96213215"},
    {"name":"hakim","number": "+-98553457"},
    {"name":"fathia","number": "+-94703051"},
    ]


namee=input("gibe me name:")
for i in peaple:
   if i["name"]==namee:
      number=i["number"]
      print(f"found{number}")
      break
else:
  print("not found")

File: phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct {
    string names;
    string number;
    }person;

int main(void)
{
    person people[3];

    people[0].names="basem";
    people[0].number="+216-96213215";
    people[1].names="fathia";
    people[1].number="+216-94703051";
    people[2].names="saleh";
    people[2].number="+216-98553457";



string name=get_string("Name: ");

for (int i=0;i<4;i++)
{
    if (strcmp(people[i].names,name)==0)
    {
    printf("Found%s\n",people[i].number);
    return 0;
    }
}

printf("Not found\n");
return 1;

}
